docs/tutorial.md

```diff
@@ -349,6 +349,137 @@ If the deployment is complete, `plan.status` in the returned JSON
+a `ServiceSpecification`.  These are the deployment and recovery plans.  
+The deployment plan is concerned with rolling out service components for
+ the first time.  The recovery plan is concerned with defining the 
+necessary operations which must be performed when transient or permanent
+failures are encountered.
+
+Let us examine the deployment plan of the example data-store service.
+Executing the following command:
+
+```bash
+$ curl -H "Authorization: token=$(dcos config show core.dcos_acs_token)" \
+          "$(dcos config show core.dcos_url)service/data-store/v1/plan"
+```
+
+generates the following output:
+
+```json
+{
+	"phases": [{
+		"id": "31a3ca99-bdbd-45dd-a8f0-cc93feade61e",
+		"name": "meta-data",
+		"blocks": [{
+			"id": "2a0db529-095b-4b63-8c56-075aa442c898",
+			"status": "COMPLETE",
+			"name": "meta-data-0",
+			"message": "Block: meta-data-0",
+			"has_decision_point": false
+		}, {
+			"id": "99210582-cdfb-4b63-95e9-19600f117dc7",
+			"status": "COMPLETE",
+			"name": "meta-data-1",
+			"message": "Block: meta-data-1",
+			"has_decision_point": false
+		}],
+		"status": "COMPLETE"
+	}, {
+		"id": "927ac279-b969-4106-9089-40264ae15dc4",
+		"name": "data",
+		"blocks": [{
+			"id": "efcf9dc0-488f-4f9e-ae0c-1ac16c401e0a",
+			"status": "COMPLETE",
+			"name": "data-0",
+			"message": "Block: data-0",
+			"has_decision_point": false
+		}, {
+			"id": "a0e0905b-d4da-4cec-84b9-461a60a92818",
+			"status": "COMPLETE",
+			"name": "data-1",
+			"message": "Block: data-1",
+			"has_decision_point": false
+		}, {
+			"id": "fa85a843-e6eb-4074-b353-875197a18adb",
+			"status": "COMPLETE",
+			"name": "data-2",
+			"message": "Block: data-2",
+			"has_decision_point": false
+		}],
+		"status": "COMPLETE"
+	}],
+	"errors": [],
+	"status": "COMPLETE"
+}
+```
+
+The first thing to notice is that the `TaskSet`s defined in a
+`ServiceSpecification` are mapped to deployment Phases. The data-store
+example service has `TaskSet`s encapsulating `meta-data` and `data`
+Tasks in that order.  The deployment plan above has the respective
+`meta-data` and `data` Phases.
+
+In its default configuration the 2 `data` Tasks and 3 `meta-data` Tasks
+are launched.  We can see that a Block has been created encapsulating 
+each of those tasks.  The names of Blocks map to the names of the
+`TaskSpecification` objects within a `TaskSet`.  The status of a Block
+can be in one of 3 states.  It may be `PENDING`, `IN_PROGRESS`, or 
+`COMPLETE`.
+
+By default all Phases and the Blocks within them are rolled out
+serially.  Each Block must reach a `COMPLETE` state before the
+next Block may be started.  Beyond this automatic behavior, it is often 
+desirable to manually pause/resume a deployment.  The
+`has_decision_point` fields above indicate whether a user has indicated 
+a desire to pause a deployment at a particular Block.
+
+A Plan's execution may be paused or continued by POST commands executed
```

Comment:
continue/interrupt doesn't seem very useful if the only strategy is INSTALL.  If I want to manage my deployment, I probably want to deploy with a non-automatic strategy.